recipes/msix/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+            raise ConanInvalidConfiguration("Only clang is supported on Linux")
+        if self.settings.os == "Linux" and (self.options.crypto_lib == "crypt32"):
+            raise ConanInvalidConfiguration("crypt32 is not available on Linux")
+        if self.settings.os == "Linux" and self.options.shared:
+            self.options.fPIC = True
+        if self.settings.os != "Android" and self.options.xml_parser == "javaxml":
+            raise ConanInvalidConfiguration("javaxml is supported only for Android")
+        if self.settings.os != "Macos" and self.options.xml_parser == "applexml":
+            raise ConanInvalidConfiguration("applexml is supported only for MacOS")
+        if self.settings.os != "Windows" and self.options.xml_parser == "msxml6":
+            raise ConanInvalidConfiguration("msxml6 is supported only for Windows")
+        if self.options.pack:
+            if self.settings.os == "Macos":
+                if not self.options.use_external_zlib:
+                    raise ConanInvalidConfiguration("Using libCompression APIs and packaging features is not supported")
+                if self.options.xml_parser and (self.options.xml_parser != "xerces"):
+                    raise ConanInvalidConfiguration("Xerces is the only supported parser for MacOS pack")
+            if not self.options.use_validation_parser:
+                raise ConanInvalidConfiguration("Packaging requires validation parser")
+        if self.options.xml_parser == "xerces":
+            self.options["xerces-c"].char_type = "char16_t"
+            self.options["xerces-c"].shared = False
+        tools.check_min_cppstd(self, "14")
+
+    def requirements(self):
+        if self.settings.os == "Linux" and not self.options.skip_bundles:
+            self.requires("icu/68.2")
+        if self.settings.os != "Windows" and not self.options.crypto_lib:
+            self.requires("openssl/1.0.2t")
+        if self.options.crypto_lib == "openssl":
+            self.requires("openssl/1.0.2t")
+        if self.options.use_external_zlib:
+            self.requires("zlib/1.2.11")
+        if self.options.xml_parser == "xerces":
+            self.requires("xerces-c/3.2.3")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("msix-packaging-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
+        if self.settings.os == "Windows":
+            self.cpp_info.system_libs = ["runtimeobject"]
+            if not self.options.crypto_lib or self.options.crypto_lib == "crypt32":
```

Comment:
crypto_lib option can never be false.
```suggestion
            if self.options.crypto_lib == "crypt32":
```